gocd/templates/bash/saas-ddog-health-check.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+/devinfra/scripts/checks/datadog/monitor_status.py --dry-run=true \
+  113296727 \
+  42722121
```

Comment:
This can be merged as it's a dry-run and won't block but once we've figured out what Datadog monitors to actually use, we should swap these values out and disable dry-run